packages/fxa-auth-server/scripts/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Small nit that's probably too much trouble to go and redo, to retain git history we might could've renamed these files with `git mv`. Seems like you either did that or git picked up on it with some other files but not all of them. If there's not a way to keep that history at this point without redoing most of them, then meh.